src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -102,14 +217,21 @@ def _handle_authentication_response(self):
         # do token request
         args = {
             'code': authn_resp['code'],
-            'redirect_uri': self.client.registration_response['redirect_uris'][0],
+            'redirect_uri':
+                self.client.registration_response['redirect_uris'][0]
```

Comment:
I really actually dislike all of the way this handles redirect urls.  As a user I'd like to be able to set this in the config and use an object to know what redirect URL I'm asking for.